src/utils/rabbitmq.py

```
@@ -33,12 +37,12 @@ async def consume_rabbitmq(connector: SCSCBotConnector):
                     if reply_to:
-                        print(result)
+                        logger.info(f"info_type=consume_rabbitmq ; {result}")
                         await channel.default_exchange.publish(
                             aio_pika.Message(
                                 body=json.dumps({"correlation_id": correlation_id, "result": result}).encode(),
                                 correlation_id=correlation_id
                             ),
                             routing_key=reply_to
                         )
-                except Exception as e: print(f"Error: {e}")
+                except Exception as e: logger.error(f"err_type=consume_rabbitmq ; Error: {e}")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)